homepage.css

```diff
@@ -0,0 +1,399 @@
+/* Styles sheet for the home page or any page with shared styles */
+/*General CSS*/
+* {
+	padding: 0;
+	margin: 0;
+	box-sizing: border-box;
+	font-family: 'Josefin Sans', sans-serif;
+}
+
+h2 {
+	color: #edffef;
+	font-size: 40px;
+	text-transform: capitalize;
+	margin-top: 0px;
+	margin-bottom: 20px;
+	text-align: center;
+}
+
+/*Navigation Bar CSS*/
+header {
+	width: 100%;
+	height: 80px;
+	display: flex;
+	align-items: center;
+	justify-content: space-between;
+	padding: 0 100px;
+	background: #acb4b5;
+	text-align: center;
+}
+
+/*Logo section CSS*/
+.logocontainer {
+	display: flex;
+	align-items: center;
+	justify-content: center;
+	margin-bottom: 20px;
+  height: 200px;
+}
+
+.logo {
+	width: 300px;
+	height: auto;
+	margin-right: 10px;
+}
+
+.logocontainer h3 {
+  font-size: 25px;
+}
+
+/*Featured Recipe Section CSS*/
+.featuredrecipes {
+	width: 100%;
+	padding: 30px 0px;
+	background-color: #213032;
+}
+
+.featuredrecipes h3 {
+	color: #edffef;
+	font-size: 40px;
+	margin-bottom: 10px;
+	margin-top: 10px;
+	text-align: center;
+}
+
+
+/*Featured recipes carousel - also on helpful tips container*/
+.container {
+	max-width: fit-content;
+	width: 40%;
+	position: relative;
+	margin-left: auto;
+	margin-right: auto;
+}
+
+.container i
+	{
+	top: 50%;
+	height: 50px;
+	width: 50px;
+	cursor: pointer;
+	font-size: 1.25rem;
+	position: absolute;
+	text-align: center;
+	line-height: 50px;
+	background: #fff;
+	border-radius: 50%;
+	box-shadow: 0 3px 6px rgba(0, 0, 0, 0.23);
+	transform: translateY(-50%);
+	transition: transform 0.1s linear;
+}
+
+.container i:first-child {
+	left: -100px;
+}
+
+.container i:last-child {
+	right: -100px;
+}
+
+.container .carousel {
+	display: grid;
+	grid-auto-flow: column;
+	grid-auto-columns: calc((100% / 3) - 12px);
+	overflow-x: auto;
+	gap: 1px;
+	border-radius: 10px;
+	scroll-behavior: smooth;
+	scrollbar-width: none;
+	position: relative;
+}
+
+.carousel::-webkit-scrollbar {
```

Comment:
This will only target webkit browsers such as Chrome and Safari. Is this intended?

Reply:
Hey Fred, no it wasn't.  My apologies - the webkit-scrollbar part of this has now been removed as per today's lesson